iceoryx_hoofs/cmake/Config.cmake.in

```diff
@@ -1,4 +1,4 @@
-# Copyright (c) 2020 by Robert Bosch GmbH All rights reserved.
+# Copyright (c) 2022 by Apex.AI Inc. All rights reserved.
```

Comment:
Why was the original copyright holder removed?

Reply:
It was a mistake I will revert it